cloudpickle/cloudpickle.py

```diff
@@ -944,27 +949,39 @@ def __reduce__(cls):
 
 
 def _fill_function(*args):
-    if len(args) == 5:
+    """Fills in the rest of function data into the skeleton function object
+
+    The skeleton itself is create by _make_skel_func().
+    """
+    if len(args) == 2:
+        func = args[0]
+        state = args[1]
+    elif len(args) == 5:
         # Backwards compat for cloudpickle v0.4.0, after which the `module`
-        # argument  was introduced
-        updated_args = args[:-1] + (None, args[-1],)
-        return _fill_function_internal(*updated_args)
+        # argument was introduced
+        func = args[0]
+        keys = ['globals', 'defaults', 'dict', 'closure_values']
+        state = dict(zip(keys, args[1:]))
+        state['module'] = None
```

Comment:
Is it ok to set `__module__` to None? Or is it preferrable to skip it entirely?

Reply:
Good catch. I meant to change that.